src/lib/receiver.js

```diff
@@ -170,13 +176,13 @@ function createReceiver (opts) {
   /**
    * Generate shared secret for Pre-Shared Key (PSK) transport protocol.
    *
-   * @return {Object} Object containing destination address and shared secret
+   * @return {PSKParams}
    */
   function generateSharedSecret () {
     const token = base64url(hmacHelper.getPskToken())
     return {
-      destination_account: getAddress() + '.' + pskReceiverId + token,
-      shared_secret: base64url(hmacHelper.getPskSharedSecret(token))
+      destinationAccount: getAddress() + '.' + pskReceiverId + token,
```

Comment:
Hmm, the snake case here wasn't random, I was thinking that you might use these values in other applications, so the field names should be standardized. It's obviously not a big deal, it's easy enough to change it manually if another implementation expects `destination_account`.

We decided to use snake case for all protocols. The question is whether JSONifying these values and copying them into a config file somewhere constitutes a "protocol".